src/index.css

```diff
@@ -28,27 +29,42 @@ body {
   place-items: center;
   min-width: 320px;
   min-height: 100vh;
+  font-family:"Merriweather", serif;
+  font-style: normal;
+  font-weight: 400;
 }
 
 h1 {
```

Comment:
This will affect all h1 classes. Maybe this should be added to your module.css instead